src/hevm/hevm.cabal

```diff
@@ -12,7 +13,7 @@ description:
 homepage:
   https://github.com/dapphub/dapptools
 license:
-  AGPL-3
+  AGPL-3.0-only
```

Comment:
Is this automated?

Reply:
not sure, I guess the cabal file was regenerated in commit 27dbc97c0bbba34907c115abf5e9cea4821fa019